server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -33,8 +33,33 @@
                 }
                 angular.extend(vm.blog, blog);
             });
+
+            if (config.output && config.output._id) {
+                outputsService.get({id: config.output._id}, function(output) {
+                    if (!angular.equals(config.output, output)) {
+                        config.output = output;
+                        applyOutputStyle();
+                    }
+                })
+            }
+        }
+
+        function fixBackgroundImage(style) {
+            if (style['background-image']) {
+                style['background-image'] = 'url(' + style['background-image'] + ')';
```

Comment:
this need to be remove actually, It is not in the release.
also themes are not ECMAScript 6